playwright/tests/homework-practice.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { test, expect } from "@playwright/test";
+
+test.describe("Test Practice Automation  HTML Form", () => {
+  test.beforeEach('Go To HTML Home Page Test Practice', async({page}) => {
+     
+     await page.goto("https://practicetestautomation.com/practice-test-login/");
+  });
+
+test("Positive LogIn test", async ({ page }) => {    
+         
+    //Type username student into Username field
+    const usernameInput = page.locator("xpath=//input[@name='username']");
+    await expect(usernameInput).toBeVisible();
+    await usernameInput.fill("student");
```

Comment:
About username and password you can try to export it as object and reuse the properties:
Interface LoginData = {
   username: string;
   password: string;
}

const loginData: LoginData = {
   username:"strudent",
   password: "Password123"
}